src/remix.ts

```diff
@@ -0,0 +1,34 @@
+import type { AppLoadContext } from '@remix-run/cloudflare'
+import type { Context } from 'hono'
+
+export type GetLoadContext = (args: {
+  request: Request
+  context: {
+    // Relaxing the type definition
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    cloudflare: any
+  }
+}) => AppLoadContext
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const defaultGetLoadContext = ({ context }: any) => {
+  return {
+    ...context,
+  }
+}
+
+export const createGetLoadContextArgs = (c: Context) => {
+  return {
+    context: {
+      cloudflare: {
+        env: c.env,
```

Comment:
I wasn't sure about this one. Do we want `c.env` which will return

```
{
      "MY_VAR": "My Valuez",
      "ASSETS": {},
      "eventContext": {
        "request": {},
        "functionPath": "/",
        "params": {},
        "data": {},
        "env": {
          "MY_VAR": "My Valuez",
          "ASSETS": {}
        }
      }
}
```

or `c.env.eventContext.env`, which would only return

```
        {
          "MY_VAR": "My Valuez",
          "ASSETS": {}
        }
```

Reply:
This function is also used in the dev server: https://github.com/yusukebe/hono-remix-adapter/pull/6/files#diff-34d76a96faa930c4b9909c055beb422e601e2267f72ce12d6f15a1c2cc6ea914R23

It should be `c.env` to get `MY_VAR` correctly since it does not have `eventContext`.